azure-pipelines.yml

```diff
@@ -254,3 +254,23 @@ jobs:
       continueOnError: true
       condition: not(succeeded())
 
+- job: RichCodeNav_Indexing
+  pool:
+    name: NetCorePublic-Pool
+    queue: BuildPool.Windows.10.Amd64.VS2017.Open
+  variables:
+    EnableRichCodeNavigation: true
+  timeoutInMinutes: 200
+  condition: and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranchName'], 'master'))
+
+  steps:
+    - script: eng/cibuild.cmd -configuration Debug -prepareMachine
+      displayName: Build
```

Comment:
Which mode is the indexer being ran here? Is this capturing the command line args and running project-at-a-time or is this passing in the entire solution and letting us do parallelization?

Reply:
It is running the compiler-invocation mode that uses command line args